nuget.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think this file is necessary?

Reply:
> I think it makes more sense to only centralize the common used packages? OxyPlot for example isn't used by the library.

For the new CPM type, it is not recommended to put version definitions inside the projects. I believe that this is a good method to have "one source of truth" about what dependencies we use. This facilitates not only updates, but also dependabot and, in general, verification of what our project is using.

Reply:
I meant this entire file doesn't seem needed, not this line :)

Reply:
If you change your VS settings that's your own responsibility, the root folder is already cluttered (and with this even more) so I'd like to avoid unnecessary files.

Reply:
I'm glad you mention it :D
I just wanted to suggest going into a more .NET Foundation style repository and arranging it more like this:

```
.
├── .github
├── scripts
├── src
│   ├── Aga.Controls
│   ├── InpOut
│   ├── LibreHardwareMonitor
│   ├── LibreHardwareMonitorLib
│   └── WinRing0
├── README.md
└── ...
```

Reply:
Yeah, that would be better, but would break all PRs, so I'd like to avoid that change (for now). If nuget.config is removed then this looks good to merge.